test/integration/crud/misc_cursors.test.js

```diff
@@ -1671,7 +1671,8 @@ describe('Cursor', function () {
     }
   });
 
-  it('removes session wheen cloning a find cursor', function (done) {
+  // TODO(NODE-4988): Need to fix cursor.clone in mongodb-legacy
+  it.skip('removes session wheen cloning a find cursor', function (done) {
```

Comment:
just two tests here

Reply:
fixed up.. these weren't actually checking for what the title says I think 🤔